### PR TITLE
UI: Update Consul UI colors to use HDS colors

### PR DESCRIPTION
**Pull request by ellacai**
### Description
- Updating Consul UI to use Hashicorp design system colours
- Phase 0 of Consul UI unification

### Testing & Reproduction steps
- click things, break stuff

### Links
[Jira](CC-4034)

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [x] not a security concern

### CODE REVIEW

File: ui/packages/consul-ui/app/components/consul/server/card/skin.scss
Comment (ellacai) [inline ui/packages/consul-ui/app/components/consul/server/card/skin.scss:25]: the figma specifies the bg color should be `red-010`, not `red-050`, so i'll follow the design and change the value here

**Comment by WenInCode** [comment]
@jherschman one thing I haven't converted is the code mirror colours. I see you had a few in the figma, but there are apparently several [custom colours declared here](https://github.com/hashicorp/consul/blob/514fb25a6fff38322398860e1b157d866c45ad81/ui/packages/consul-ui/app/components/code-editor/skin.scss#L4-L24). Mind giving that a look and seeing if we want to convert to HDS colors for those? Feels less crucial, but I'm happy to do so given you are able to provide guidance on mapping them over.

File: ui/packages/consul-peerings/app/components/consul/peer/components.scss
Comment (jherschman) [inline ui/packages/consul-peerings/app/components/consul/peer/components.scss:71]: This badge should use:

Text: Color Variable: Foreground/Warning-On-Surface (AKA Amber 300, #9E4B00)
Background color seems accurate here

File: ui/packages/consul-peerings/app/components/consul/peer/components.scss
Comment (jherschman) [reply ui/packages/consul-peerings/app/components/consul/peer/components.scss:67]: image
how it appears right now

File: ui/packages/consul-peerings/app/components/consul/peer/components.scss
Comment (jherschman) [inline ui/packages/consul-peerings/app/components/consul/peer/components.scss:61]: image

I believe this is the value getting overridden causing the terminated badge to not have a background color  (screenshot below)

**Review by jherschman — CHANGES_REQUESTED** [review]
You did a crazy good job of getting the colors mapped properly props to you and Ella.... Took me forever to find small inconsistencies but the majority of the app is 🔥 . I highlighted anything I could find minus the badges that we discussed in slack, but happy to include here as well if that would helpful.

**Review by valeriia-ruban — APPROVED** [review]
That's a huge piece of work! I'm happy you and Ella cleaned up so many style files!

Hope merging with buttons won't be too painful 🤞 
LGTM 🚀 🥇 ✅

**Comment by ellacai** [comment]
Awesome stuff Tyler!! 💯 Thanks so much for all your hard work 
I don't think I can approve since I'm an author but it looks good to me 👍

File: ui/packages/consul-ui/app/components/filter-bar/skin.scss
Comment (jherschman) [inline ui/packages/consul-ui/app/components/filter-bar/skin.scss:4]: Looks like there is an extra bottom border showing on the peers filter bar... 
note: this is currently like this in production so not because of color updates, so happy to add a separate ticket to remove the additional boarder here if we want to tackle separately)
image
image

**Review by jherschman — CHANGES_REQUESTED** [review]
Some small nav color updates added in screenshot to provide context on updates Let me know if you have any questions! Everything else looking good to go!! 
image

File: ui/packages/consul-ui/app/components/filter-bar/skin.scss
Comment (WenInCode) [reply ui/packages/consul-ui/app/components/filter-bar/skin.scss:4]: Done ✅

**Comment by WenInCode** [comment]
@jherschman I've updated with the feedback you gave in slack, and fixed the border issue in peers. I rebased this against main to fix the conflicts so it will be good to go once it gets the approval.

**Review by jherschman — APPROVED** [review]
Huge amount of work and all of it looks great! Thank you @WenInCode and @ellacai for tackling this enormous ticket. This is a great foundational improvement and will make the rest of our updates for unification that much easier moving forward! LGTM 🚀 🥳

**Pull request merged**
